docs/userguide/searchspace.md

```diff
@@ -232,79 +222,17 @@ The function [`SearchSpace.from_product`](baybe.searchspace.core.SearchSpace.fro
 ### Constructing from a Dataframe
 
 [`SearchSpace.from_dataframe`](baybe.searchspace.core.SearchSpace.from_dataframe) constructs a search space from a given dataframe.
-Due to the ambiguity between discrete and numerical parameter choices, this function requires a parameters list with pre-defined parameter objects, unlike its subspace counterparts, where this list is optional.
+Due to the ambiguity between discrete and continuous parameter representations when identifying parameter ranges based only on data, this function requires that the appropriate parameter definitions be explicitly provided. This is different for its subspace counterparts [`SubspaceDiscrete.from_dataframe`](baybe.searchspace.discrete.SubspaceDiscrete.from_dataframe) and [`SubspaceContinuous.from_dataframe`](baybe.searchspace.continuous.SubspaceContinuous.from_dataframe), where a fallback mechanism can automatically infer minimal parameter specifications if omitted.
 
 ```python
 from baybe.searchspace import SearchSpace
 
-params = [
-    NumericalDiscreteParameter(name="x0", values=[1, 2, 3]),
-    NumericalDiscreteParameter(name="x1", values=[4, 5, 6]),
-    NumericalContinuousParameter(name="x2", bounds=(6, 9)),
-]
-
-df = pd.DataFrame(
-    {
-        "x0": [2, 3],
-        "x1": [5, 4],
-        "x2": [9, 7],
-    }
-)
-searchspace = SearchSpace.from_dataframe(df=df, parameters=params)
+p_cont = NumericalContinuousParameter(name="c", bounds=[0, 1])
+p_disc = NumericalDiscreteParameter(name="d", values=[1, 2, 3])
+df = pd.DataFrame({"c": [0.3, 0.7], "d": [2, 3]})
+searchspace = SearchSpace.from_dataframe(df=df, parameters=[p_cont, p_disc])
 ```
 
-Since one of the provided parameters is continuous, this creates a hybrid space.
```

Comment:
why remove the "print"?

Reply:
and we cant add an explicit print?

Reply:
I guess there must be a way somehow – would come in handy in several places. But unlike our examples, the user guide is not `executed`

Reply:
right, forgot that
Im still not convinced that removing is useful.

Clearly there is the danger that this print gets outdated... so what?

There is a greater benefit of anything that even appears in the userguide: make users and LLMs aware that it exists

Reply:
I agree that having it in there is generally good and also with the LLM part. So let me simply bring it back for now (in its updated form) and then we can:
* Figure out how to solve this problem in general, also for other user guide pages
* Once the search space changes are completed, i.e. immutability/metadata issue completely fixed, overdue cleanup of search space code / hypothesis (which I think will be my work item after pareto), we do an overhaul of the search space guide, which is very limited currently and could definitely benefit from a restructuring

**Note:** In order to bring it back, I had to modify the printing configuration a little bit. @AVHopp: can you also double-check if this makes sense?

Reply:
@AVHopp: note that I did NOT change the default. Just made `None` possible as input to allow arbitrarily widths. The reason was that the parameter list would otherwise not be fully visible in the search space dump. So it has only a local effect